scripts/observation_view.py

```diff
@@ -24,7 +24,9 @@
 import numpy as np
 import datetime
 
-import sbp_piksi as ids
+#from sbp.deprecated import SBP_MSG_OLD_OBS TODO should we keep around deprecated messages?
```

Comment:
Do we want to preserve deprecated messages and continue to support deprecated message paths like here?